packages/components/src/internal/events.ts

```diff
@@ -46,6 +46,10 @@ export function isSelectAll(event: React.KeyboardEvent<any>): boolean {
     return isEvent(event) && event.keyCode === KEYS.A && (event.ctrlKey || event.metaKey);
 }
 
+export function isFillDown(event: React.KeyboardEvent<any>): boolean {
```

Comment:
nit: Consider refactoring these events to:

```ts
function isEvent(event: React.SyntheticEvent<any>): boolean {
    return event !== undefined && event !== null;
}

function isMetaKeyEvent(event: React.KeyboardEvent<any>, keyCode: number): boolean {
    return isEvent(event) && event.keyCode === keyCode && (event.ctrlKey || event.metaKey);
}

export const isCopy = (event: React.KeyboardEvent<any>): boolean => isMetaKeyEvent(event, KEYS.C);
export const isFillDown = (event: React.KeyboardEvent<any>): boolean => isMetaKeyEvent(event, KEYS.D);
export const isPaste = (event: React.KeyboardEvent<any>): boolean => isMetaKeyEvent(event, KEYS.V);
export const isSelectAll = (event: React.KeyboardEvent<any>): boolean => isMetaKeyEvent(event, KEYS.A);
```

Reply:
I like it. Thanks for the patch.